src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -173,13 +161,17 @@ public Void call() throws Exception {
                 query.select(r)
```

Comment:
use getDeviceBy Id  method or just use IsmDeviceApi.this.em.find(DeviceEntity.class, vs.getMgrId());.

Whenever you are doing operations on the manager we need to use the mgr id not vs.getId()(this is the OSC id)

org.osc.sdk.manager.element.VirtualSystemElement.getMgrId()

Reply:
OK, taken care...

Reply:
OK